bookdown/06-special-tasks/06-cost-sens.Rmd

```diff
@@ -53,7 +57,7 @@ Our goal is to find a model which minimizes the costs (and thereby maximizes the
 ### A First Model
 
 For our first model, we choose an ordinary logistic regression (implemented in the add-on package `r mlr_pkg("mlr3learners")`).
-We first create a classification task, then resample the model using a 10-fold cross validation and extract the resulting confusion matrix:
+We first create a classification task, then re-sample the model using a 10-fold cross validation and extract the resulting confusion matrix:
```

Comment:
I think "resample" is correct here. We also have it as a function name and I've so far never seen "re-sample" in any article or so?